electron_src/main.js

```diff
@@ -53,7 +53,8 @@ function createWindow() {
     minWidth: 600,
     webPreferences: {
       contextIsolation: true,
-      preload: `${__dirname}/preload.js`
+      preload: `${__dirname}/preload.js`,
+      webSecurity: false
```

Comment:
any problem if web security equals to true? is it related to eletron upgrade?

Reply:
The cookie op will fail due to samesite if websecurity is true.